target_chains/ethereum/contracts/contracts/pyth/Pyth.sol

```diff
@@ -521,6 +581,37 @@ abstract contract Pyth is
         }
     }
 
+    function getPriceFeedsIndex(
+        bytes32[] memory priceIds,
+        bytes32 targetPriceId
+    ) private pure returns (uint index) {
+        uint k = 0;
+        for (; k < priceIds.length; k++) {
+            if (priceIds[k] == targetPriceId) {
+                break;
+            }
+        }
+        return k;
+    }
+
+    function setPriceFeedsInfo(
```

Comment:
can we rename it to `fillPriceFeedsInfo`? `set` might be a bit confusing (as it's used for state setters)